<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenCalc</class>
 <widget class="QMainWindow" name="FenCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>122</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="num2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>61</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="equalsButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>96</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLabel" name="result">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>65</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="num1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>61</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="operations">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>76</width>
      <height>29</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>*</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>/</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>%</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>505</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
